Sources/nest/Commands/UninstallCommand.swift

```diff
@@ -27,6 +27,15 @@ struct UninstallCommand: AsyncParsableCommand {
         let targetCommand = info[commandName, default: []].filter { command in
             command.version == version || version == nil
         }
+
+        if info[commandName, default: []].isEmpty {
+            logger.error("🪹 \(commandName) doesn't exist.", metadata: .color(.red))
+            Foundation.exit(1)
+        } else if let version, targetCommand.isEmpty {
+            logger.error("🪹 \(commandName) (\(version)) doesn't exist.", metadata: .color(.red))
+            Foundation.exit(1)
+        }
```

Comment:
I guess you divided the if statements for error log.
However, it's a bit diffcult to understand when the program exits here.
Could you merge the if statements and switch the error message inside the if statement?

Reply:
I have been fixing it; is this modification what you intended? [8bcc418](https://github.com/mtj0928/nest/pull/9/commits/8bcc418fea238a53b1123f8a324f0704d7dcc5df)